REFERENCE.md

```diff
@@ -7,30 +7,31 @@
 ### Classes
 
 * [`k8s`](#k8s): Sets up a Kubernetes instance - either as a node or as a server
-* [`k8s::install::kubeadm`](#k8s--install--kubeadm): Installs the kubeadm binary
-* [`k8s::install::kubectl`](#k8s--install--kubectl): Installs the kubectl binary
-* [`k8s::node`](#k8s--node): Installs a Kubernetes node
-* [`k8s::node::kube_proxy`](#k8s--node--kube_proxy): Sets up a on-node kube-proxy instance
-* [`k8s::node::kubectl`](#k8s--node--kubectl): Installs the kubectl binary
-* [`k8s::node::kubelet`](#k8s--node--kubelet): Installs and configures kubelet
-* [`k8s::repo`](#k8s--repo): Handles repositories for the container runtime
-* [`k8s::server`](#k8s--server): Sets up a Kubernetes server instance
-* [`k8s::server::apiserver`](#k8s--server--apiserver): Installs and configures a Kubernetes apiserver
-* [`k8s::server::controller_manager`](#k8s--server--controller_manager): Installs and configures a Kubernetes controller manager
-* [`k8s::server::etcd`](#k8s--server--etcd): Sets up an etcd cluster node
-* [`k8s::server::etcd::setup`](#k8s--server--etcd--setup): Installs and configures an etcd instance
-* [`k8s::server::resources`](#k8s--server--resources): Generates and deploys standard Kubernetes in-cluster services
-* [`k8s::server::scheduler`](#k8s--server--scheduler): Installs and configures a Kubernetes scheduler
-* [`k8s::server::tls`](#k8s--server--tls): Generates the necessary Kubernetes certificates for a server
+* [`k8s::install::kubeadm`](#k8sinstallkubeadm): Installs the kubeadm binary
```

Comment:
looks like you generated it with puppet-strings 2. Please update your puppet-strings to version 3 (that's the version also used in the CI).

Reply:
Interesting, this was done from a fresh clone and bundle install, so apparently there's a `puppet-strings` dependency that's too lenient.